Tests/BasicsTests/ObservabilitySystemTests.swift

```diff
@@ -133,45 +136,46 @@ final class ObservabilitySystemTest: XCTestCase {
         testDiagnostics(collector.diagnostics, problemsOnly: false) { result in
             do {
                 let diagnostic = result.check(diagnostic: "error", severity: .error)
```

Comment:
Consider using `try #require()` on these declarations so the values aren't optional.